subsystem-drive/implementations/rules-engine.hpp

```diff
@@ -32,11 +43,13 @@ namespace sjsu::drive
                 commands.speed = std::clamp(commands.speed, -kMaxSpeed, kMaxSpeed);
                 //sjsu::LogInfo("Specified speed is too fast... clamping speed");
             }
+
             heartbeat_.IncrementHeartbeatCount();
             return commands;
         }
 
     private:
         sjsu::common::Heartbeat heartbeat_;
+        const int clamp_angle = 12;
```

Comment:
```suggestion
        static constexpr int clamp_angle = 12;
```
rename clamp_angle to kMaxClampAngle as well...

Reply:
Should it be static "int" constexpr kMaxClampAngle? Because other wise it doesn't recognize it as a declared variable.